logger/glog/glog_test.go

```diff
@@ -428,6 +432,309 @@ func TestLogBacktraceAt(t *testing.T) {
 	}
 }
 
+func TestExtractTimestamp(t *testing.T) {
+	preffix := fmt.Sprintf("%s.%s.%s.log.", "geth_test", "sampleHost", "sampleUser")
+	cases := []struct {
+		name     string
+		fileName string
+		expected string
+	}{
+		{"valid INFO", preffix + "INFO.20171202-132113.2841", "20171202-132113"},
+		{"valid WARNIG", preffix + "WARNING.20171202-210922.13848", "20171202-210922"},
+		{"valid gzipped", preffix + "WARNING.20171202-210922.13848.gz", "20171202-210922"},
+		{"extra long filename", preffix + "WARNING.20171202-210922.13848.gz.bak", "20171202-210922"},
+		{"to short filename", preffix + "WARNING.20171202-21092", ""},
```

Comment:
s/to/too

Reply:
I need to `:set spell` before next commit ;)